.github/workflows/ci.yaml

```diff
@@ -42,11 +46,8 @@ jobs:
       - name: Install deps (with cache)
         run: pnpm install
 
-      - name: Typecheck
-        run: pnpm typecheck
+      - name: Build, lint, test, and typecheck
```

Comment:
Idk if this should be in this PR, but fine 😅

Reply:
I tihnk it's leftovers from when i tested a bunch of stuff and got build errors on the examples - I can remove it and make a new one

Reply:
Don't worry about it